castanet/counter.py

```diff
@@ -121,8 +135,16 @@ def sum_cast_dict(cast_dict):
     return total
 
 def assignment_count(cast_dict):
-    """A function for counting the number of assignmnet"""
-    # An example of an assignmnet is x = y
+    """A function for counting the number of assignmnet
```

Comment:
This should read `A function for counting the number of assignment statements in a Python program.`